### DISCUSSION TITLE
Change Display within Text/number box

### DISCUSSION THREAD

**Discussion by MTPGthb** [mProjectsCode/obsidian-meta-bind-plugin#272] [Q&A] [answered]
This is mainly for aesthetic, Is there anyway to display the Metadata in question within the displayed box? 

For example, 

`INPUT[text(showcase):breakfast]` //displays a box with "Text"

is there anyway to change the display "text" to "breakfast"

**Comment by mProjectsCode** [comment, marked as answer]
I am assuming you are referring to the placeholder. The placeholder can be changed with the [placeholder argument](https://www.moritzjung.dev/obsidian-meta-bind-plugin-docs/reference/inputfieldarguments/placeholder/).

**Reply by MTPGthb** [reply]
By chance do you know the Custom CSS that changes the toggle to a checkbox, the same within the "ITS" theme.
